docs/installation.rst

```diff
@@ -36,15 +36,18 @@ most small to medium setups.
 Prepare virtual environment (recommended)
 =========================================
 
-Choose an installation path, here `/home/john/ihatemoney`.
+Choose an installation path, here the current user's home directory (`~`).
 
 Create a virtual environment::
 
-    python3 -m venv /home/john/ihatemoney
+    cd ~
+    mkdir ihatemoney
+    cd ihatemoney
+    python3 -m venv .
```

Comment:
That's more generic indeed, but a bit too long:
```suggestion
    python3 -m venv ~/ihatemoney
    cd ~/ihatemoney
```

The `cd` can be omitted if we keep an absolute path for the following activation.

Reply:
Sure. I have a slight preference for doing things in the current directory, with relative paths, but I don't feel strongly about it.